lib/device-log/ios-log.js

```diff
@@ -209,6 +211,7 @@ class IOSLog {
       }
     }
     this.proc = null;
+    IOSLog.cachedIDeviceSysLogs[this.udid] = null;
```

Comment:
this object structure is in conflict with `_getIDeviceSysLogProcess`m since there `cmd` is used as the key

Reply:
Good catch.